client/app/components/pages/License.vue

```diff
@@ -1,46 +1,46 @@
-<style>
-
-</style>
+<style></style>
 
 <template>
-  <div style="width: 500px; margin-left:auto;margin-right:auto;margin-top: 100px">
-    The MIT License (MIT) <br/><br/>
-
-    Copyright (c) <2014> <copyright Marthlab></copyright> <br/><br/>
+  <div
+    style="
+      width: 500px;
+      margin-left: auto;
+      margin-right: auto;
+      margin-top: 100px;
+    "
+  >
+    The MIT License (MIT) <br /><br />
+
+    Copyright (c) <2014> <copyright Marthlab></copyright> <br /><br />
 
     Permission is hereby granted, free of charge, to any person obtaining a copy
-    of this software and associated documentation files (the "Software"), to deal
-    in the Software without restriction, including without limitation the rights
-    to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-    copies of the Software, and to permit persons to whom the Software is
-    furnished to do so, subject to the following conditions:<br/><br/>
+    of this software and associated documentation files (the "Software"), to
```

Comment:
what is the new line length we're establishing? I suggest we keep it consistent with 120 chars, as we do in other projects

Reply:
Default `printWidth` is set at 80, I can update to 120

https://prettier.io/docs/en/rationale.html#:~:text=%7D)()%3B-,Print%20width,meet%20the%20specified%20print%20width.

Reply:
I think for these things it'll be useful to add a `prettier.config.js` file

Reply:
I have updated rules for quotes + printWidth
Added config file with multiple different parsers
Only js, html, css, json and vue files should be updated -- no minified js, other formats can be added if needed